### ISSUE TITLE
Getting capped at 90 samples per label when creating dataset

### ISSUE THREAD

**Issue by chrishadiono** [NVIDIA/DIGITS#2098]
Hi, 
I am making a dataset where I have hundreds of images per label. However, when I create a dataset with this data in DIGITS it only includes 90 of them maximum per label. I have attached an image of what I mean here. I tried using the mnist data and did not experience this cap using the exact same settings. Originally I thought it was because my images are each around 30kb each and maybe Digits cannot handle large sizes, but I also tried reducing the jpeg quality of the images very low so they were each 3-4kb each (like the mnist example) and still was capped at 90. Any help would be greatly appreciated!

**Comment by chrishadiono** [comment]
Update: Looking through the digits/tools/create_db.py I think the problem could be in this function which specifies 100 as the image count 

`def _calculate_batch_size(image_count, is_hdf5=False, hdf5_dset_limit=None,
                          image_channels=None, image_height=None, image_width=None):
    """
    Calculates an appropriate batch size for creating this database
    """
    if is_hdf5 and hdf5_dset_limit is not None:
        return min(100, image_count, hdf5_dset_limit / (image_channels * image_height * image_width))
    else:
        return min(100, image_count)`

**Comment by jw3** [comment]
Yeah I have encountered this as well.  Looks like the cap is specified here

https://github.com/NVIDIA/DIGITS/blob/master/digits/tools/parse_s3.py#L372
